robot_modeling/DQ_SerialManipulator.m

```diff
@@ -45,16 +60,17 @@
 %
 % Contributors to this file:
 %     Bruno Vihena Adorno - adorno@ufmg.br
+%     Juan Jose Quiroz Omana - juanjqogm@gmail.com
 
 % TODO: Remove the virtual joints. Instead of helping, they cause a lot of
 % confusion, specially among those trying to learn the library. The
 % affected methods are: FKM and Jacobian.
 
-classdef DQ_SerialManipulator < DQ_Kinematics
+classdef (Abstract) DQ_SerialManipulator < DQ_Kinematics
     properties        
-        theta,d,a,alpha;
-        convention;        
-        effector;
+        %theta,d,a,alpha;
+        %convention;        
+        curr_effector_;
```

Comment:
Since the main goal of this PR was to update the Matlab version with respect to the version of C++, I implemented the variables using the same name as in C++. I do not know the motivation for _curr_effector__. @mmmarinho do you have any comments about it?

Reply:
@juanjqo  It should be effector_ in both languages. Style-wise it should be pared with the setter and getter methods. 

curr_effector_ is probably a remnant of an older implementation.

Reply:
@dqrobotics/developers I modified  curr_effector_ by effector_.